sw/host/opentitantool/opentitantool_derive/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "opentitantool_derive"
+version = "0.1.0"
+authors = ["lowRISC contributors"]
+edition = "2018"
+
+[lib]
+proc-macro = true
```

Comment:
You madman.

Reply:
"You coward, you servant, you blind man... back to the front!"

[Disposable Heroes](https://www.youtube.com/watch?v=jtNrx1c3Xh8) by Metallica.